firebase/web_scrape/Library.py

```diff
@@ -0,0 +1,79 @@
+# Sriram Srinivasan
```

Comment:
Please remove your name from this pull request. Once you do that, I can go ahead and merge this PR.